glass/shapes.py

```diff
@@ -31,7 +31,7 @@
 import numpy as np
 
 if TYPE_CHECKING:
-    from numpy.typing import ArrayLike, NDArray
+    from numpy.typing import ArrayLike, NDArray  # pragma: no cover
```

Comment:
I've said this in a separate PR, but I think we should do this https://coverage.readthedocs.io/en/latest/excluding.html#advanced-exclusion